eslint/babel-eslint-parser/README.md

```diff
@@ -91,6 +91,20 @@ module.exports = {
 };
 ```
 
+**Monorepo configuration**
+
+This configuration is usefull for monorepo as it avoids to repeat the configuration of babel and eslint on every package.
+
+```
+{
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "babelOptions": {
+      "rootMode": "upward"
+    }
+  }
+}
+```
```

Comment:
I would use a js config example as in other parts of the docs.

Reply:
Yeah you're right, I've tested and `.eslintrc.js` are working as well as `.eslintrc`